docs/content/set-up-new-project.md

```diff
@@ -0,0 +1,78 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+
+-----
+
+## Setting up a new project
+
+Follow the [Setup and installation guide](../setup-and-installing.md) to make sure you have everything installed.
+
+To use the SpatialOS GDK for Unity in a new project, you need to set up your project manifest, and then set up world initialization. Your new project must have the [same structure as a SpatialOS project](https://docs.improbable.io/reference/latest/shared/reference/project-structure).
+
+### Set up your project manifest
+Add the following dependencies to the `packages` manifest located inside `workers/unity/Packages/manifest.json`:
+```
+{
+  "dependencies": {
+    "com.improbable.gdk.core": "file:<path-to-the-gdk>/workers/unity/Packages/com.improbable.gdk.core",
+    "com.improbable.gdk.legacy": "file:<path-to-the-gdk>/workers/unity/Packages/com.improbable.gdk.legacy",
```

Comment:
This no longer exists and should now be `com.improbable.gdk.buildSystem`

Reply:
ah, I fixed it in the other doc, but forgot about it here

Reply:
@zeroZshadow which bit exactly should be "com.improbable.gdk.buildSystem"?